src/mimir_coordinator.py

```diff
@@ -259,6 +260,11 @@ def _build_blocks_storage_config(self) -> Dict[str, Any]:
         }
 
     def _build_s3_storage_config(self, s3_config_data: _S3ConfigData) -> Dict[str, Any]:
+        # Remove the scheme from the s3 endpoint for the Mimir config
+        parsed_url = urlparse(s3_config_data.endpoint)
+        scheme = "%s://" % parsed_url.scheme
+        s3_config_data.endpoint = parsed_url.geturl().replace(scheme, "", 1)
```

Comment:
Nitpick: `replace` is more than `strip_prefix` (which isn't available on py3.8).
How about:
```python
>>> url = "scheme://localhost:9090"
>>> re.sub(fr'^{urlparse(url).scheme}://', '', url)
'localhost:9090'
```